assets/scss/_variables_project.scss

```diff
@@ -80,7 +80,7 @@ h2, h3 {
   }
   .td-sidebar-link {
     &__page {
-      color: $gray-900;
+      //color: $gray-900;
```

Comment:
So, I guess this affects the color of the _unselected_ page links in the sidebar. It looks like the old `gray-900` variable was resolving to the [`#222222` color](https://www.color-hex.com/color/222222).  By removing this here, we are dropping back to using [`#495057`](https://www.color-hex.com/color/495057) which is just a lighter shade of gray.

![image](https://github.com/medic/cht-docs/assets/8441903/df7898c3-f6b9-4957-86aa-17a99d85bf45)

Personally, the lighter shade is fine with me, but if we wanted to stick with what we had before, we could just hard-code the color value (either way we should remove the commented-out code):

```suggestion
      color: #222222;
```